madmin/static/vars/vars_parser.json

```diff
@@ -449,7 +449,7 @@
 					"settings": {
 						"type": "option",
 						"require": "true",
-						"values": "true|false",
+						"values": "All|Unknown Only",
```

Comment:
Not possible - please let this unchanged. the other stuff is ok

Reply:
Ill make some edits tomorrow if i think of it. Also what you mentioned has been improved with some other PR recently  :)